reana-ui/src/components/WorkflowProgressCircleBar.js

```diff
@@ -0,0 +1,103 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2024 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import PropTypes from "prop-types";
+import styles from "./WorkflowProgressCircleBar.module.scss";
```

Comment:
Another cosmetic observation. Whilst the workflow is running, we are displaying (correctly) green progress bar part for those jobs that have finished. However, on the worklfow list page, this gives too much "green" for a workflow that is "blue" because it is still running.

Perhaps we may want to show the progress circle in blue if the status of the workflow is running? In this way it'll be easy to distinguish directly which workflows are finished and which are still running, and the colour consistency may look better in that way.

![blue](https://github.com/reanahub/reana-ui/assets/517546/dc1569a5-7c68-420d-a7bf-053a04f0792a)

Reply:
Progress bar is now blue, with solid stroke for finished workflows and dashed stroke for running ones.